compose/docker-compose.yaml

```diff
@@ -1,20 +1,22 @@
-version: '3.7'
+version: "3.7"
+name: studies-management
 services:
   ego-api:
-    image: 'overture/ego:3.4.0'
+    image: "ghcr.io/overture-stack/ego:edge"
```

Comment:
this should remain using the latest published version (fixed tag) to avoid adding unaccounted complexity